assets/js/main.js

```diff
@@ -370,4 +370,58 @@ $(function() {
   };
   anchors.add().remove('.no-anchor');
   anchors.add('body.terms .terms-row .tos-table__legal p strong');
-});
\ No newline at end of file
+});
+
+/* Add cookie prompt functionality */
+(function(){
+  var linkText = "cookie policy";
+  var linkUrl = "/cookie-policy/";
+  var dismissBtn = "OK";
+  var msg = "By using this website you agree to our " + linkText.link(linkUrl);
+
+
+  //check if cookie already exists
+  if(document.cookie){
+    var cookieString = document.cookie;
+    var cookieList = cookieString.split(";");
+    // if cookie named GruntyCookie is found, return
+    for(x = 0; x < cookieList.length; x++){
+      if (cookieList[x].indexOf("GruntyCookie") != -1){
+        console.log(cookieList[x].indexOf("GruntyCookie"));
+        return
+      };
+    }
+  }
+
+  // variables
+  var docRoot = document.body;
+  var cookieModalC = document.createElement("div");
+  cookieModalC.setAttribute("id", "gruntyCookie");
+  var cookieModalCp = document.createElement("p");
+  cookieModalCp.innerHTML = msg;
+  // var cookieModalCText = document.createTextNode(msg);
+
+  //dismiss button
+  var cookieModalCclose = document.createElement("button");
+  var cookieModalCloseText = document.createTextNode(dismissBtn);
+  cookieModalCclose.setAttribute("id", "cookieModalClose");
+  cookieModalCclose.setAttribute("class", "btn btn-primary");
+  cookieModalCclose.appendChild(cookieModalCloseText);
+  cookieModalCclose.addEventListener("click", closeCookie, false);
+
+
+  //add to DOM
+  //cookieModalCp.appendChild(cookieModalCText);
+  cookieModalC.appendChild(cookieModalCp);
+  cookieModalC.appendChild(cookieModalCclose);
+  docRoot.appendChild(cookieModalC);
+
+  cookieModalC.classList.add("cookieModalCBeginAnimate");
+
+  function closeCookie(){
+    var cookieExpire = new Date();
+    cookieExpire.setFullYear(cookieExpire.getFullYear() +2);
+    document.cookie="GruntyCookie=1; path=/; expires=" + cookieExpire.toGMTString() + ";";
```

Comment:
A reusable `setCookie()` method would be cleaner. Here's another example: https://stackoverflow.com/a/24103596/483528